gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/explore.proto

```diff
@@ -33,6 +33,7 @@ message ExploreRequest {
 
   string space_id = 22;
   optional ContextOption context_option = 23;
+  bool fetch_total = 24;
```

Comment:
Why do we need this?

The total count will be retrieved using a COUNT(*) query with the same set of filters. The UI can execute this query in parallel if we want to display the total count in the UI.

Internally, we will implement it the similar way by executing two parallel queries.  But, it will be executed for each pagination request too.

We have overloaded the traces/spans API in the past and observed that for every pagination, it runs this query. So, if the UI fires the total query separately, it will be executed only once.

Reply:
This is needed to get the count of groupby. We won't be calculating again and again. UI will not fetch for total all the time, if total is not required. It will make separate query for total

Reply:
actually found the way fixing this PR